packages/ui/src/components/molecules/Dialog/UsbTroubleShootDialogBox.tsx

```diff
@@ -0,0 +1,181 @@
+import React, { FC, ReactNode } from 'react';
+import { styled } from 'styled-components';
+
+import {
+  ArrowButton,
+  Button,
+  Flex,
+  LangDisplay,
+  Typography,
+  TypographyColor,
+} from '../../atoms';
+import { BulletList } from '../BulletList';
+import { GoldenArrowList } from '../GoldenArrowList';
+import { WaitingDiv, WaitingDivType } from '../WaitingDiv';
+
+import { DialogBoxBody, DialogBoxFooter, DialogBoxHeader } from '.';
+
+const InnerContainer = styled.div`
+  padding-top: 32px;
+  padding-left: 40px;
+  padding-right: 40px;
+  max-height: 58vh;
+  overflow-y: auto;
+`;
+
+export interface UsbTroubleShootDialogBoxProps {
+  title?: string;
+  title2?: string;
+  subtitle?: string;
+  bulletList?: string[];
+  messageBoxList?: Record<string, string>[];
+  heading?: string;
+  image: React.ReactElement;
+  children?: ReactNode;
+  isLoading?: boolean;
+  loadingText?: string;
+  footer?: ReactNode;
+  goldenArrowList?: any[];
+  disablePrev?: boolean;
+  disableNext?: boolean;
+  onNext: React.MouseEventHandler<HTMLButtonElement>;
+  onPrevious: React.MouseEventHandler<HTMLButtonElement>;
+  isFirstDialog?: boolean;
+}
+export const UsbTroubleShootDialogBox: FC<UsbTroubleShootDialogBoxProps> = ({
```

Comment:
This component is almost a duplicate of `GuidedFlowDialogBox`. Prefer reusing the same component to avoid duplicate code.

Reply:
created a `reusable` component as `CustomFloeDialogBox`